### PR TITLE
Added post-login action multipliers

### PR DESCRIPTION
**Pull request by arizonatribe**
Added post-login action multipliers, to reduce the three ways a user may gain authentication (login, token refresh, or parsing and validating the token from storage) down to one action that dependent components may hook onto.

### CODE REVIEW

File: src/ducks.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import {createRow} from 'attadux'
+import auth from './components/auth/ducks'
+
+/**
+* Creating a row of ducks makes more sense when there's many ducks:
+* @example createRow(hueyDuck, deweyDuck, louieDuck, donaldDuck, scroogeMcDuck)
+*/
+
+export default createRow(auth)
```
Comment (AlexMaskovyak) [inline src/ducks.js]: Ha, getting your ducks in a row. Hilarious!

Comment (gsiegman) [reply src/ducks.js]: Missing daisyDuck, PR denied.

Comment (AlexMaskovyak) [reply src/ducks.js]: Launchpad McQuack?

Comment (arizonatribe) [reply src/ducks.js]: Oh yeah

Comment (arizonatribe) [reply src/ducks.js]: Darkwing, Gosalyn, Webigail, Howard

**Pull request merged**